### DISCUSSION TITLE
Need help with Docusaurus plugin

### DISCUSSION THREAD

**Discussion by TheNinth7** [jgraph/drawio#4046] [Q&A]
Good morning,

I am trying out this draw.io plugin for Docusaurus: https://github.com/xiguaxigua/docusaurus-plugin-drawio

It seems to work great, the only issue I have is that zooming in does not really work as I expect. First, once zoomed in, horizontal scrolling is only possible with keyboard arrows, which is not that intuitive. Also, while the graphic stays within its limit of the page, the toolbar is extended, which does not look that nice:

![Screenshot - 16_12_2023 , 07_56_22](https://github.com/jgraph/drawio/assets/118298758/d87c4f5a-93d7-4c9e-a380-936fa6885cc9)

Does anyone know how to fix the toolbar when zooming in? And if there is a way to enable scrolling per mouse?

Thanks for your help!

Regards, Robert

**Comment by davidjgraph** [comment]
Never heard of Docusaurus, have you tried asking the author of the plugin?

**Reply by TheNinth7** [reply]
No, not yet, thought I'll ask in the community first, I guess the author gets flooded with such requests. But I now created issue https://github.com/xiguaxigua/docusaurus-plugin-drawio/issues/13 in the plugin's repository

**Reply by hinorashi** [reply]
@davidjgraph here you are: https://github.com/facebook/docusaurus, kinda famous in static site generator.

**Reply by davidjgraph** [reply]
@hinorashi what's your point?
